OJP/OJP_Trips.xsd

```diff
@@ -604,24 +604,24 @@
 	</xs:complexType>
 	<xs:complexType name="PathGuidanceStructure">
 		<xs:annotation>
-			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information </xs:documentation>
+			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element name="PathGuidanceSection" type="PathGuidanceSectionStructure" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>one or more path guidance sections that build the trip Leg</xs:documentation>
+					<xs:documentation>One or more PATH GUIDANCE sections that build the LEG. The sections may need to be rather short to incororpate the necessary guidance in a detailed level. This may depend a lot on the MODE and on the type of guidance that is needed.</xs:documentation>
```

Comment:
First sentence: "One or more PATH GUIDANCE sections that form the LEG."
Second sentence: I don't really get what this is supposed to mean "to incororpate the necessary guidance in a detailed level". This should be reworded.
Third sentence: "This may depend heavily on the MODE and the type of guidance required."

Reply:
rewritten